src/drive/components/LoadMore.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import classnames from 'classnames'
+import { Button } from 'cozy-ui/react'
+import styles from '../styles/table'
+
+const LoadMore = props => (
```

Comment:
It would be cool to display a spinner instead of the button when the files are loaded, that's what we do in Photos

Reply:
Okay, I will add something to cover this feature.

Reply:
Voilà https://github.com/cozy/cozy-drive/pull/560/files#diff-cbb9c8196fa98b59f2362793f4b86dc0R35